tests/integration/test.replication.js

```diff
@@ -293,13 +321,21 @@ adapters.forEach(function (adapters) {
     });
 
     it('Test pull replication with many conflicts', function (done) {
+      if (testUtils.isSyncGateway()) {
+        return done();
+      }
       var remote = new PouchDB(dbs.remote);
 
-      var numRevs = 200; // repro "url too long" error with open_revs
+      var numRevs = 10; // repro "url too long" error with open_revs
       var docs = [];
       for (var i = 0; i < numRevs; i++) {
-        var rev =  '1-' + PouchDB.utils.uuid(32, 16).toLowerCase();
-        docs.push({_id: 'doc', _rev: rev});
+        // var rev =  '1-' + PouchDB.utils.uuid(32, 16).toLowerCase();
+        docs.push({
+          _id: 'doc', 
+          _revisions: {
+            start : 1, ids : [PouchDB.utils.uuid(32, 16).toLowerCase()]
+          }
+        });
```

Comment:
Let's go ahead and include both `_rev` and `_revisions` when we do `bulkDocs`; the behavior is undefined otherwise, and I know PouchDB handles it weirdly compared to CouchDB. Much clearer if we just add both.